src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -279,13 +279,14 @@ def _write_pes_to_nwbfile(
         else:
             timestamps_cleaned = timestamps
```

Comment:
This block is now unnecessary/unused with the timestamps fix

Reply:
Can you elaborate?

Reply:
Ah, I see, I think the fix is wrong. Let me check.

Reply:
Should be fixed now.